pkg/meta/config.go

```diff
@@ -59,6 +59,7 @@ type Format struct {
 	Inodes           uint64 `json:",omitempty"`
 	EncryptKey       string `json:",omitempty"`
 	KeyEncrypted     bool   `json:",omitempty"`
+	ChaCha20         bool   `json:",omitempty"`
```

Comment:
I think we should have `EncryptAlgo = chacha20-rsa`, so we can support other algorithm later.

Reply:
done